FueledUtils/UIExtensions.swift

```diff
@@ -127,4 +153,10 @@ public extension UIImage {
 		}
 		return image.resizableImage(withCapInsets: self.capInsets, resizingMode: self.resizingMode).withRenderingMode(self.renderingMode)
 	}
+
+	public func resized(offset: CGPoint = .zero, size: CGSize) -> UIImage? {
```

Comment:
I think we need to add a documentation comment here to be very clear when it's appropriate to use. This is much slower than system resize functionality when you just draw a large image in a small view but will have better quality.

Reply:
I see, I mean I know how that works, I just wasn't sure what you were referring to.
This method is for the cases specifically where that isn't possible (hmm `NSTextAttachment` hmm, for example). I think the method is explicit enough (considering the swift 3 convention) that this will create a new image, and could potentially take a long time to do so.
I have never seen anyone trying to resize an image into a `UIImageView` in that manner, and I believe that we'll catch it through review if it ever happens ;) so I don't think adding a comment is necessary.